cmd/launcher/launcher.go

```diff
@@ -157,6 +158,12 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 		}
 	}
 
+	// init osquery instance history
+	err = osqueryInstanceHistory.InitHistory(db)
+	if err != nil {
+		return errors.Wrap(err, "error initializing osquery instance history")
+	}
```

Comment:
Talking aloud... This feels odd here, and not implicit in `NewInstance`, or in the runner setup. But I think I see why... I assume it can't be implicit, because the History struct is global, and we don't pass the DB to the runner.

Honestly, I think this is probably pointing to issues with the existing code. And this is probably a pragmatic choice here.